### ISSUE TITLE
Selection box appears to offset the token's x/y values

### ISSUE THREAD

**Issue by Tyaeth** [trioderegion/dnd5e-helpers#79]
Hi there,

I was fiddling around with my combat settings earlier and noticed that when an actor attacks diagonally when another actor is also nearby, it's counting the other adjacent creature as half-cover.

Sample:
![FoundryVTT_WzYXwsmuVM](https://user-images.githubusercontent.com/6688230/112765489-70528c80-8fdb-11eb-86b5-d45cded0eb78.png)

Attacking diagonally around a wall seems to work properly though:
![FoundryVTT_HiUHZGwvvp](https://user-images.githubusercontent.com/6688230/112765556-adb71a00-8fdb-11eb-8f44-398724c19302.png)

**Comment by trioderegion** [comment]
I appreciate the example images.

And yes, I need to revisit how I am handling token/tile collisions as they dont use the 4 corner vision rules like walls can -- its center to center and my "hitboxes" are a tad too big for this specific case.

In your second image, do you have 4 corner vision enabled? or Centerpoint (foundy) vision?

**Comment by Tyaeth** [comment]
In both samples I have Four Corner Vision enabled. 

When using Centerpoint (Foundry) vision, the second image produces 3/4 covert.

**Comment by trioderegion** [comment]
Ok great. Things are at least working as well as I expect them to. Let me try reducing the token/tile hitboxes some in the short term.

**Comment by trioderegion** [comment]
addressed in 1.10.3 (released). This should work better :)

**Comment by Tyaeth** [comment]
Thanks for the fix! Seems to be working great for me now :)

**Comment by Tyaeth** [comment]
Just wanted to provide an additional heads up - I was still experiencing the issue in some really weird situations where there definitely shouldn't have been any cover (like targeting a creature directly north of me when there was a creature to the immediate east somehow still providing cover for it), and at that point it just seemed way too farfetched to think that my original issue was causing it. 

I was still experiencing the issue when I disabled all of my modules except for D&D5e Helpers and Easy Target, yet everything seemed fine when I had only D&D5e Helpers activated and did the targeting manually. After a bit of trial and error I realized that the Alt key shows a highlight around all tokens that shows their alignment towards the selected token, and that this happens naturally when pressing the Alt+Left Click combination required to use Easy Target. 

From what I can tell this highlight actually expands the size of the hitbox, which I think was causing my original issue. Here's a quick gif sample where I target as I would normally using Easy Target vs when I shake it around a little bit to remove the highlight.
![QQ2AscoTWX](https://user-images.githubusercontent.com/6688230/112772462-f207e180-8ffe-11eb-9163-7b2fedb76b9d.gif)

I'm not sure if that's something worth accounting for, but I thought I'd report back with what I found at the very least in case someone else runs into the same issue.
